src/main/java/com/manufacturing/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,109 @@
+package com.manufacturing.dao.jdbc;
+
+import com.manufacturing.dao.ManufacturerDao;
+import com.manufacturing.lib.Dao;
+import com.manufacturing.model.Manufacturer;
+import com.manufacturing.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    private static final String MANUFACTURER_NAME = "manufacturer_name";
+    private static final String MANUFACTURER_ID = "manufacturer_id";
+    private static final String ORIGIN = "origin";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, origin) VALUE (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("This data can't be added to table", e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long manufacturerId) {
+        String query = "SELECT * FROM manufacturers WHERE manufacturer_id = ? AND deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, manufacturerId);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                Manufacturer manufacturer = new Manufacturer(resultSet.getString(MANUFACTURER_NAME),
+                        resultSet.getString(ORIGIN));
+                manufacturer.setId(manufacturerId);
```

Comment:
This logic can be extracted to the utility method and reused in getAll(), for example. Also, getString() and getLong() methods can cause NullPointerException. Remember about [checklist](https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/05_JBDC_checklist.html)